docker/pytorch-aarch64/scripts/build-torchdata.sh

```diff
@@ -0,0 +1,39 @@
+#!/usr/bin/env bash
+
+# *******************************************************************************
+# Copyright 2022 Arm Limited and affiliates.
+# Copyright 2022 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+# SPDX-License-Identifier: Apache-2.0
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+# *******************************************************************************
+
+set -euo pipefail
+
+source python3-venv/bin/activate
```

Comment:
Do we need to activate the python environment here? We don't usually put in the script explicitly https://github.com/ARM-software/Tool-Solutions/blob/master/docker/pytorch-aarch64/scripts/build-scipy.sh. I

Reply:
That's a fair point. With `VIRTUAL_ENV` set, and the `PATH` updated accordingly, there should be no need to activate the venv via the script. As you noted, we don't do it for any of the other steps.

Reply:
Just to confirm - removing this `source` line didn't cause me any issues.